src/fmu/dataio/dataio.py

```diff
@@ -404,10 +403,7 @@ class ExportData:
     _rootpath: Path = field(default_factory=Path, init=False)
 
     def __post_init__(self) -> None:
-        assert isinstance(self.config, dict)
         logger.info("Running __post_init__ ExportData")
-        logger.debug("Global config is %s", prettyprint_dict(self.config))
```

Comment:
The simple exporters now provide the config as a `GlobalConfiguration` model, which made this `assert` started failing. It was only added there to satisfy mypy as the `prettyprint_dict` below requires `dict` input. This debug print of the entire config have been a bit polluting the logs when trying to debug, so I just removed it now.